content/docs/manage/facilities.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Facilities"
+description = "Manage all stores"
+date = 2022-05-17T16:20:00+00:00
+updated = 2022-05-17T16:20:00+00:00
+draft = false
+weight = 2
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+The Facilities list is available only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view all stores, and manage their properties.
+
+## Viewing Facilities
+
+Choose `Manage` > `Facilities` in the navigation panel.
+
+![goto: facilities](/docs/manage/images/goto_facilities.png)
+
+You will be presented with a list of stores in your system.
+
+![facility list](/docs/manage/images/facilities.png)
+
+The list of Stores is divided into 5 columns:
+
+| Column       | Description                      |
+| :----------- | :------------------------------- |
+| **Code**     | The code for the store           |
+| **Name**     | Store name                       |
+| **Supplier** | Whether this store is a supplier |
+| **Customer** | Whether this store is a customer |
+| **Donor**    | Whether this store is a donor    |
+
+<div>
+You will also see an icon like this (<img src="/docs/replenishment/images/is_msupplystoreicon.png" alt="Store" style="width:auto">) next to the store code - meaning this is an mSupply store.
+</div>
+
+The list can display a fixed number of stores per page. On the bottom left corner, you can see how many stores are currently displayed on your screen.
+
+![Pagination: showing](/docs/distribution/images/os_list_showing.png)
+
+If you have more stores than the current limit, you can navigate to the other pages by clicking on the page number or using the right of left arrows (bottom right corner).
+
+![Pagination: navigating](/docs/distribution/images/os_list_pagenumbers.png)
+
+You can also select a different number of rows to show per page using the option at the bottom right of the page.
+
+![Rows per page](/docs/introduction/images/rows-per-page-select.png)
+
+### Searching facilities
+
+You can filter the list of facilities by name or by code. This can be useful if you're looking for one particular store!
+
+In the search bar in the top left of your screen, type some (or all) of a store name or store code. The list will now contain all matching facilities:
+
+![Facilities filter](/docs/manage/images/facilities_filter.png)
+
+## Editing store properties
+
+To edit the properties of a store in the list, click on it. This will open a new `Edit Facility` window.
+
+<div class='tip'>
+If you see a message like this when you open the Edit Facility window, that means you have no store properties configured! To set these up, please contact support.
```

Comment:
They're configured in the `Settings` (server admin) page... are we fine with "please contact support"?

Reply:
yep. all roads lead to support..